subgraphs/smolverse/schema.graphql

```diff
@@ -47,3 +48,47 @@ type Token @entity {
   image: String
   owner: User
 }
+
+type Farm @entity {
+  id: ID!
+  name: String!
+  stakedTokens: [StakedToken!]! @derivedFrom(field: "farm")
+}
+
+type StakedToken @entity {
+  id: ID!
+
+  "Internal for tracking pending claims"
+  _pendingClaimId: String
+
+  token: Token!
+  farm: Farm!
+  owner: User!
+  stakeTime: BigInt!
+  claims: [Claim!]!
+}
+
+type Random @entity {
+  id: ID!
+
+  "Internal for tracking associated claim"
+  _claimId: String
+}
+
+type Seeded @entity {
+  id: ID!
+
+  "Internal for tracking associated random requests"
+  _randomIds: [String!]!
+}
+
+enum ClaimStatus {
```

Comment:
Not a big deal, but I prefer the enums at the top for some reason, lol. Up to you if you want to move it.

Reply:
Agree on this! Moved it to the top now